src/BinSkim.Rules/DoNotShipVulnerableBinaries.cs

```diff
@@ -80,13 +80,22 @@ private static StringToVersionMap BuildDefaultVulnerableBinariesMap()
 
         public override void Analyze(BinaryAnalyzerContext context)
         {
+            // Throw a platform unsupported exception--FileVersionInfo does not behave "correctly" on Linux.
+            BinaryParsers.PlatformSpecificHelpers.ThrowIfNotOnWindows();

```

Comment:
This is a good and clean change. Moving forward, I wonder if it'd be helpful to annotate rules with some metadata that indicates a dependency on PDBs, allowing us to move this handling somewhere in a more central place. Something to keep in mind.